src/app/landing/LandingPage.jsx

```diff
@@ -136,87 +160,138 @@ class LandingPage extends Component {
+              img={{
+                small: 'img/partners/infoxchange_mobile_logo.png',
+                large: 'img/partners/infoxchange_logo.png',
+              }}
             />
             <PartnerPicture
               link="http://minnow.io/"
               alt="Minnow logo"
-              src="img/partners/minnow_logo.png"
+              img={{
+                small: 'img/partners/minnow_mobile_logo.png',
+                large: 'img/partners/minnow_logo.png',
+              }}
             />
             <PartnerPicture
               link="http://neighborstogether.org/"
               alt="neighbors together logo"
-              src="img/partners/neighbors_together_logo.png"
+              img={{
+                small: 'img/partners/neighbors_together_mobile_logo.png',
+                large: 'img/partners/neighbors_together_logo.png',
+              }}
             />
             <PartnerPicture
               link="https://www.ncsinc.org/"
               alt="Neighborhood Coalition For Shelter logo"
-              src="img/partners/neighborhood_coalition_for_shelter_logo.png"
+              img={{
+                small: 'img/partners/neighborhood_coalition_for_shelter_mobile_logo.png',
+                large: 'img/partners/neighborhood_coalition_for_shelter_logo.png',
+              }}
             />
             <PartnerPicture
               link="https://www1.nyc.gov/site/dhs/index.page"
               alt="nyc dhs logo"
-              src="img/partners/nyc_logo.png"
+              img={{
+                small: 'img/partners/nyc_mobile_logo.png',
+                large: 'img/partners/nyc_logo.png',
+              }}
             />
             <PartnerPicture
               link="http://www.law.nyu.edu/"
               alt="nyu law logo"
-              src="img/partners/nyu_law_logo.png"
+              img={{
+                small: 'img/partners/nyu_law_mobile_logo.png',
+                large: 'img/partners/nyu_law_logo.png',
+              }}
             />
             <PartnerPicture
               link="https://opencollective.com/streetlives"
               alt="Open Collective logo"
-              src="img/partners/open_collective_logo.png"
+              img={{
+                small: 'img/partners/open_collective_mobile_logo.png',
+                large: 'img/partners/open_collective_logo.png',
+              }}
             />
             <PartnerPicture
               link="https://openreferral.org/"
               alt="open referral logo"
-              src="img/partners/openreferral_logo.png"
+              img={{
+                small: 'img/partners/openreferral_mobile_logo.png',
+                large: 'img/partners/openreferral_logo.png',
+              }}
             />
             <PartnerPicture
               link="https://holyapostlessoupkitchen.org/"
               alt="Holy Apostles logo"
-              src="img/partners/holy_apostles_logo.png"
+              img={{
```

Comment:
An alternative solution, that wouldn't require so many changes to the code (and won't require further changes if we need to change this again), would be to do something like what native apps do: put different sizes of the same picture in a directory, or some standardized naming format, pass to the component just the base name (e.g. holy_apostles_logo.png) and then let the component handle the logic of choosing a version.

But not necessary, if you think it would take non-trivial work.

Reply:
Good idea, I'll change the filenames and implement this.